Patches test10 to avoid conflict with _a, _b and constants defined in internal header files.
--- tests/c/test10.c.orig	2021-09-20 07:52:33.000000000 -0500
+++ tests/c/test10.c	2023-03-22 16:37:09.000000000 -0500
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -102,8 +102,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
--- tests/c/test10_f32.c.orig	2021-10-08 06:46:05.000000000 -0500
+++ tests/c/test10_f32.c	2023-03-22 16:37:09.000000000 -0500
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -102,8 +102,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
--- tests/c/test10_f32_i32.c.orig	2021-10-08 06:46:05.000000000 -0500
+++ tests/c/test10_f32_i32.c	2023-03-22 16:37:09.000000000 -0500
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -102,8 +102,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
--- tests/c/test10_f32_i64.c.orig	2021-10-08 06:46:05.000000000 -0500
+++ tests/c/test10_f32_i64.c	2023-03-22 16:37:09.000000000 -0500
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -102,8 +102,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
--- tests/c/test10_i32.c.orig	2021-10-08 06:46:05.000000000 -0500
+++ tests/c/test10_i32.c	2023-03-22 16:37:09.000000000 -0500
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -102,8 +102,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
--- tests/c/test10_i64.c.orig	2021-10-08 06:46:05.000000000 -0500
+++ tests/c/test10_i64.c	2023-03-22 16:37:09.000000000 -0500
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -102,8 +102,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
--- tests/c/test10_mem.c.orig	2017-07-10 15:25:25.000000000 +0100
+++ tests/c/test10_mem.c	2021-02-08 10:30:35.000000000 +0000
@@ -14,8 +14,8 @@
  *  You should have received a copy of the GNU Lesser General Public License
  *  along with MED.  If not, see <http://www.gnu.org/licenses/>.
  */
-#define _a 0.446948490915965
-#define _b 0.091576213509771
+#define _A 0.446948490915965
+#define _B 0.091576213509771
 #define _p1 0.11169079483905
 #define _p2 0.0549758718227661
 
@@ -74,8 +74,8 @@
 
   /* Constantes */
 
-  med_float gscoo1_1[12] = { 2*_b-1, 1-4*_b, 2*_b-1, 2*_b-1, 1-4*_b,
-                             2*_b-1, 1-4*_a, 2*_a-1, 2*_a-1, 1-4*_a, 2*_a-1, 2*_a-1 };
+  med_float gscoo1_1[12] = { 2*_B-1, 1-4*_B, 2*_B-1, 2*_B-1, 1-4*_B,
+                             2*_B-1, 1-4*_A, 2*_A-1, 2*_A-1, 1-4*_A, 2*_A-1, 2*_A-1 };
   med_float wg1_1[6] = { 4*_p2, 4*_p2, 4*_p2, 4*_p1, 4*_p1, 4*_p1 };
 
   med_int   nval1_1= 1*6; /*1 valeurs et 6 points de gauss par valeur */
